.github/workflows/create-tag.yml

```diff
@@ -131,3 +135,13 @@ jobs:
 
           gh repo set-default ${{ github.repository }}
           gh release create "$TAG" --verify-tag --draft --title "$TAG" -F "$TEMPFILE"
+
+      - name: Create PR updating dependencies
+        if: ${{ github.event.inputs.update_deps == 'true' }}
+        run: |
+          make update-dependencies
+          git checkout -b temporal-data/update-dependencies
```

Comment:
maybe it should use a unique branch name each time? if the branch isn't deleted from the last one the push will fail. or push with --force

Reply:
I added the base commit hash as part of branch to make it somewhat unique.